packages/@react-spectrum/table/src/Resizer.tsx

```diff
@@ -14,16 +16,41 @@ import {VisuallyHidden} from '@react-aria/visually-hidden';
 interface ResizerProps<T> {
   column: GridNode<T>,
   showResizer: boolean,
-  triggerRef: RefObject<HTMLDivElement>
+  triggerRef: RefObject<HTMLDivElement>,
+  onMovedResizer: (e: MoveMoveEvent) => void
```

Comment:
nitpick: perhaps onResizerMove? or onMoveResizer? Since this is called in onMove should it be present tense?